recipes/fortnet/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "fortnet" %}
+{% set version = "0.7.2" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+{% set origin = "vanderhe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ origin }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 77f9477231491af7400ef74061d89149b67eae14bd8fedce690a4a79b2294c9c
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  run_exports:
+    - {{ name|lower }} * {{ mpi_prefix }}_*
+    - {{ pin_subpackage( name|lower, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - python
+    - h5py
+    - numpy
+    - cmake
+    - ninja
+    - fypp
```

```suggestion
    - cmake
    - ninja
```
We're not running any of these tests, so the python libraries are not needed?

---

As far as I can see python, h5py and numpy are needed for my [minimal tests](https://github.com/vanderhe/staged-recipes/blob/00208afb855d2ba739f123d438ea7a4d37c91815/recipes/fortnet/build.sh#L45-L55) to ensure build sanity. However, I removed the fypp dependency.

---

I didn't see those tests! Thanks for pointing them out.